presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -388,7 +388,10 @@ Property Name                             Description
 
 ``metrics_max_inferred_column``            Optionally specifies the maximum number of columns for which     ``100``
                                            metrics are collected.
-=======================================   ===============================================================   ============
+
+``read.split.target-size``                 The target size for an individual split when generating splits     ``134217728`` (128MB)
```

Comment:
It seems Presto convention does not use the prefixes from the Iceberg lib. E.g. "write.metadata.metrics.max-inferred-column-defaults" was just named "metrics_max_inferred_column" table property. While I do think the notion with prefixes is clearer, I think it's better to name it "split_target_size" for now. Please also add explanation here this correspond to `read.split.target-size` table property in Iceberg library. 

Later on, we can send a proposal to use full iceberg property names since it will affect the users.

Reply:
@hantangwangd and I have discussed this in another PR and decided that moving forward for new properties we will use the iceberg property names. We will introduce backwards-compatible property names for the properties which were already introduced and slowly phase the old names out over the next few releases. I have filed this issue to address it: https://github.com/prestodb/presto/issues/24483

Reply:
I agree that using Iceberg names is clearer. But ideally we should make changes for #24483 before directly using the new name. When and who will be working on it? Do you think you can send PR for that sooner?

Reply:
Sure, I can try to get that PR out this week. I can definitely have it in soon, but since most users won't see any changes until the 292 release, I am in the camp that it would be fine to not hold up this PR to align the old property names

Reply:
Sounds good! It should be good as long as it's before the 292 release.

Reply:
Here is a draft PR for introducing the deprecation of table property names: https://github.com/prestodb/presto/pull/24581

It is still WIP. Just needs some tests

Reply:
> Here is a draft PR for introducing the deprecation of table property names: #24581
> 
> It is still WIP. Just needs some tests

Great! Ping me when it's ready.